### ISSUE TITLE
Requests modal javascript is dead.

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/SearchWorks#3262]
I don't see this selector anywhere: https://github.com/sul-dlss/SearchWorks/blob/master/app/assets/javascripts/jquery.requestsModal.js#L138. which implies this whole javascript file can be deleted.

**Comment by jcoyne** [comment]
It looks like it was removed in https://github.com/sul-dlss/SearchWorks/pull/1075

**Comment by marlo-longley** [comment]
Yes I see a comment [here](https://github.com/sul-dlss/SearchWorks/commit/ee4c53b92b44ad91035d4524d9ee46d98fe72b32) saying:

"Removing modal since login screwed us"
